django_admin_shell/views.py

```diff
@@ -244,10 +247,20 @@ def form_valid(self, form):
     def get_context_data(self, **kwargs):
         """Add output to context"""
         ctx = super(ShellView, self).get_context_data(**kwargs)
+
+        if django.VERSION >= (3, 2):
```

Comment:
* Can we move this part to separate method `self.get_add_snippet_url()` in view: 
    * for consistency 
    * for isolation 
    * for ability  to override in future. 

```
def get_add_snippet_url(self):
    """Return url address to django admin site to section for adding saved snippet"""
    if django.VERSION >= (3, 2):
        return reverse('admin:admin_shell_savedsnippet_add')
    return reverse('admin:django_admin_shell_savedsnippet_add')
```